src/controllers/trips.controller.js

```diff
@@ -73,6 +78,8 @@ class Trip {
     });
     await mail.lineManagerNotification();
 
+    await messenger(phoneNumber, `New request initiated by ${currentUser.name}. Kindly review`);
```

Comment:
Can you do error handling. What if the user has no line manager?

Reply:
When a user has no line manager they can't even create a trip. They are redirected to profile to update and assign themselves a line manager

![image](https://user-images.githubusercontent.com/52410156/74927332-10ed7a80-53e0-11ea-9663-7e42c8dba6b2.png)